arch/xtensa/src/common/xtensa_dumpstate.c

```diff
@@ -96,10 +96,10 @@ static inline void up_showtasks(void)
 #endif
 
 /****************************************************************************
- * Name: xtensa_stackdump
+ * Name: up_stackdump
  ****************************************************************************/
 
-static void xtensa_stackdump(uint32_t sp, uint32_t stack_top)
+static void up_stackdump(uint32_t sp, uint32_t stack_top)
```

Comment:
This is an internal function, the naming convention for these functions is `arch_function`.

Reply:
Ok. I will revert this part of the change

Reply:
@Ouss4 done